Kickstarter-iOS/Views/Cells/DiscoveryPostcardCell.swift

```diff
@@ -189,6 +189,7 @@ internal final class DiscoveryPostcardCell: UITableViewCell, ValueCell {
     self.viewModel.outputs.metadataData
       .observeForUI()
       .observeValues { [weak self] data in
+        self?.metadataIconImageView.isHidden = data.iconImage == nil
```

Comment:
@justinswart you're totally right. I have no idea why this wasn't broken out into several outputs. It may have been built very early or done out of time constraint at the time before shipping. @bormind having only one side effect per signal makes it easier to test and understand exactly what is being changed (for example, label text versus just knowing that an entire piece of data changed)